src/content/overview/observability/configuration/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Configuration
+description: Learn how to configure and customize the Giant Swarm observability platform for your organization's needs.
+weight: 30
+menu:
+  principal:
+    parent: overview-observability
+    identifier: overview-observability-configuration
+last_review_date: 2025-06-30
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How do I configure the observability platform?
+  - What configuration options are available?
+  - How do I set up multi-tenancy?
+  - What can I customize in Grafana?
+  - How do I configure data ingestion?
+  - What alerting options can I configure?
+  - How do I manage access controls?
+---
+
+The Giant Swarm observability platform offers extensive configuration options to tailor the system to your organization's specific monitoring, alerting, and visualization needs. This overview covers the key areas you can configure to create a customized observability experience that fits your operational requirements.
+
+## Platform-level configuration
+
+Configure foundational platform settings that affect the entire observability stack:
+
+### Initial platform setup
+
+Work with Giant Swarm to configure core platform components:
+
+- **OIDC provider integration**: Connect your identity provider for authentication
+- **Base domain configuration**: Set up DNS and ingress for platform access
+
+## Multi-tenancy and access control
+
+**Multi-tenancy** forms the foundation of platform configuration, enabling you to:
+
+- **Isolate data** between teams, environments, or projects through tenant separation
+- **Control access** by mapping Grafana organizations to specific tenants and user groups
+- **Manage permissions** with role-based access controls (RBAC) from your identity provider
+- **Configure tenant lifecycle** including creation, management, and removal processes
+
+Multi-tenancy configuration affects all other platform components, making it essential to understand before configuring other areas.
+
+### Organization management
+
+Manage your observability platform's organizational structure through **[`GrafanaOrganization`]({{< relref "/reference/platform-api/crd/grafanaorganizations.observability.giantswarm.io" >}})** resources. Organization management enables you to:
```

Comment:
And here we should link to the organization page (which also is the multi-tenancy I guess but with Grafana Orgs as anchor maybe?